app/assets/javascripts/dashboard/dashboard_task_list_view.js

```diff
@@ -324,6 +376,18 @@ class DashboardTaskListView extends React.PureComponent<Props, State> {
     }
   }
 
+  handleLoadMoreTasks = () => {
+    this.setState(prevState => {
+      const newUnfinishedTasks = prevState.unfinishedTasks.filter(t => t.id !== task.id);
+      const newFinishedTasks = [changedTask].concat(prevState.finishedTasks);
```

Comment:
Where is `changedTask` coming from? Looks like copy&paste error from `confirmFinish`?

Reply:
In general, this code doesn't seem to match the method name?